lib/command.js

```diff
@@ -84,13 +84,21 @@ function encode() {
   var sender = this.sender;
   var target = this.target;
   var header = new Buffer(9);
+
   var content = !data || isBuffer(data) ? data : this.encode(data);
 
   if (content) length += content.length;
 
   header.writeUInt8(length, 0);
-  header.writeUInt8(sender.subnet, 1);
-  header.writeUInt8(sender.id, 2);
+
+  //For some reason panel lighting changes need to come from 253.254
+  if (this.code == 0xE14E || this.code == 0xE012) {
```

Comment:
Respectfully disagree from a usability perspective, but your repo so will proceed as requested.  Will implement a 'sendAs' routine to perform the send function, but allowing the setting of the sender.

Reply:
`Bus` class represents a virtual "HDL IP Gateway" device running in node.js environment. So it have its own subnet and id. This is why I suggesting you to create another instance if you need another sender subnet and id. 

Maybe this class should be called `Gateway` and have to be associated with a `Bus` object which represents properties of a real HDL IP Gateway and allows to send any message to it.

Now `Gateway` and `Bus` are merged in same class called `Bus` and it's now very useful when you would like to send messages from custom sender. 

The main idea is to keep things simple, this library is not intended to be a simple DSL for HDL bus, it just implements the protocol, and allows to build UDP message understandable by HDL. 

It would be better to have special classes on top of generic protocol implementation which provides simplier DSL for custom cases, like it was implemented for channels.

For example it could be done like this:

```js
const panel = bus.enviro('1.15');

enviro.backlight(100, function(err) {});
```

Reply:
agree, but can we get past this PR then look to develop further?

Reply:
Yes, right now you can create this kind of DSL on top of this library inside your code.